charts/karmada-dashboard/values.yaml

```diff
@@ -100,7 +94,8 @@ api:
       maxUnavailable: 0
       maxSurge: 50%
   ## @param apikubeconfig kubeconfig of the api
```

Comment:
could you help to update the comment for `api.kubeconfigName` and `api.kubeconfigContext`